pkg/controller/core.oam.dev/v1alpha2/application/assemble/options.go

```diff
@@ -104,20 +113,11 @@ func discoverHelmModuleWorkload(ctx context.Context, c client.Reader, assembledW
 	return nil
 }
 
-// NameNonInplaceUpgradableWorkload set workload name with component revision name to override component name.
-func NameNonInplaceUpgradableWorkload() WorkloadOption {
```

Comment:
why it's not used?

Reply:
It's re-wrote in approllout controller pkg. https://github.com/oam-dev/kubevela/blob/61d7aff01c2020c9994fac085006c34a855da6c9/pkg/controller/core.oam.dev/v1alpha2/applicationrollout/helper.go#L35

Reply:
ok